suites/nautilus/ansible/tier-0.yaml

```diff
@@ -0,0 +1,72 @@
+#===============================================================================================
+#-------------------------------------
+#-----    Smoke Test suite     ------
+#-------------------------------------
+# Conf: conf/nautilus/ansible/tier-0.yaml
+# Smoke test cases for
+#    - Install RHCS4 with lvm scenario
+#
+#===============================================================================================
+tests:
+   - test:
+      name: install ceph pre-requisites
+      module: install_prereq.py
+      config:
+        is_production: true
+      abort-on-fail: true
+
+   - test:
+      name: ceph ansible
+      polarion-id: CEPH-83571467
+      module: test_ansible.py
+      config:
+        is_mixed_lvm_configs: True
+        ansi_config:
```

Comment:
Could we correct the indentation? Our guidelines is to have 2 spaces for YAML files.